tests/mitoai_ui_tests/mitoai.spec.ts

```diff
@@ -29,7 +29,7 @@ test.describe('Mito AI Chat', () => {
     await createAndRunNotebookWithCells(page, ['import pandas as pd\ndf=pd.DataFrame({"A": [1, 2, 3], "B": [4, 5, 6]})']);
     await waitForIdle(page);
 
-    await clearMitoAIChatHistory(page);
+    await startNewMitoAIChat(page);
```

Comment:
Can we update our tests to use multiple workers again? They are pretty slow to run rn